contracts/contracts/poolBooster/PoolBoostCentralRegistry.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+
+/**
+ * @title Contract that holds all governance approved pool booster Factory
+ *        implementation deployments
+ * @author Origin Protocol Inc
+ */
+contract PoolBoostCentralRegistry is Governable {
```

Comment:
If we want a registry of pool boosters, would it be easier to actually store the pool booster addresses here, as well as emit the events here? Otherwise the backend has to listen to this contract, then add listeners for each factory in order to listen to new events.

If we had pool creation events here, then this would be all that would need to be tracked.

Each pool creation event could also emit the factory contract address that created it / submitted it.

Reply:
I am on the fence deciding if the extra complexity justifies the monitoring convenience. The way I am imagining it the necessary complexity to add would be: 
  - Central registry needs an external function to emit pool booster events. 
  - factories would need to have a reference to central registry so it can callback when a new pool booster is created. 
  - when the mentioned function is permissioned meaning it cycles through all the factories it holds in its array

Reply:
Just a mapping of factories is fine. Backend can have a list from events. Backend no longer needs to talk to each factory to find out what is going on.

Reply:
Anothe way to look at the complexity here is that the factories, of which we have a lot, and will build a lot, no longer have to have any tracking code. The amount of code, and of duplicate deployed code will go down.

Reply:
You make a good point that for monitoring this will simplify things quite significantly. Change here: https://github.com/OriginProtocol/origin-dollar/pull/2400/commits/0148ddda29170e498c4d0ad97307e2ece5d6b7b2

Reply:
Actually re-introduced the list back from mapping. As this way it is easy to iterate over all of the supported factories and call `bribeAll` on each of them: https://github.com/OriginProtocol/origin-dollar/pull/2400/commits